.travis.yml

```diff
@@ -8,18 +8,10 @@ cache:
     - vendor
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-
-env:
-  matrix:
-    - PREFER_LOWEST="--prefer-lowest"
-    - PREFER_LOWEST=""
+  - 7.2
```

Comment:
I would suggest to copy the stuff from https://github.com/JouwWeb/SlmQueue/blob/master/.travis.yml. That way you test more future versions, and the repos are more in sync :).